lib/ifquery

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python
+# Crudely emulate ifquery on systems that don't have it.
```

Comment:
i must admit, getting rid of some of the Python code, makes me a bit greedy about wanting to get rid of all of it;
that would also mean, that I'd try to replace `ifquery` with shell [ simply to reduce the dependency on Python ]

but let's leave it as is for now